.github/workflows/update-dependencies.yml

```diff
@@ -0,0 +1,48 @@
+name: "Update Dependencies"
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 16 * * 1"  # Mondays at 8am PST
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  update-dependencies:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup .NET 9
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: "9.0.x"
+
+      - name: Restore solution
+        run: ./build.sh -restore
+
+      - name: Rewrite nuget.config
+        run: dotnet new nugetconfig --force
+
+      - name: Install dotnet-outdated
+        run: dotnet tool install --global dotnet-outdated-tool
+
+      - name: Update packages
+        run: dotnet outdated --no-restore -u ./Aspire.sln
+
+      - name: Revert all changes except Directory.Packages.props
+        run: |
+          git add Directory.Packages.props
+          rm nuget.config
+          git checkout -- .
+
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@67ccf781d68cd99b580ae25a5c18a1cc84ffff1f
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+          branch: update-dependencies
+          commit-message: "[Automated] Updating package dependencies"
+          title: "[Automated] Update dependencies"
+          body: "Auto-generated update to the package dependencies. In order for this PR to be green, it will require all of the new dependencies to be mirrored to our AzDO NuGet feeds."
```

Comment:
should this comment say "but before merging revert changes to XYZ and ABC"

Reply:
or "consult Jose" :)

Reply:
Sure, I can add that as well.